src/k8s-engine/lima.ts

```diff
@@ -196,6 +197,8 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   /** Process for tailing logs */
   protected logProcess: childProcess.ChildProcess | null = null;
 
+  debug: boolean;
```

Comment:
Since this is always false via the constructor, we can simplified it to:
```suggestion
  debug = false;
```
(And drop the assignment in the constructor.)